Tests/SwiftFormatPrettyPrintTests/PrettyPrintTestCase.swift

```diff
@@ -19,20 +19,69 @@ public class PrettyPrintTestCase: XCTestCase {
 
     // Assert that the input, when formatted, is what we expected.
     if let formatted = prettyPrintedSource(input, configuration: configuration) {
-      XCTAssertEqual(
-        expected, formatted,
-        "Pretty-printed result was not what was expected",
-        file: file, line: line)
+      XCTAssertDiff(
```

Comment:
Let's revert the changes in this file before merging so that we're not duplicating this code in two places. We should move common code into a separate test-support module, and I'd also like to replace the current diffing logic (which was a quickly written temporary solution) with something more general purpose, but none of that is relevant to the change being made by this PR so it would be better to keep this change localized.

Reply:
That makes sense. I added it in because it's so much easier to spot what's wrong with a diff 😅